.github/workflows/unit-tests.yaml

```diff
@@ -18,16 +18,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
Apart from the commit SHA pinning and bumping some actions from `vX` to `vY`, this is the only _logical_ change. Happy to revert this if we want to run tests also against e.g. current - 1.